src/res/translation/translation_fr_FR.ts

```diff
@@ -738,7 +738,7 @@
     <message>
         <location filename="../../clientdlg.cpp" line="376"/>
         <source>C&amp;hat...</source>
-        <translation>&amp;Tchate...</translation>
+        <translation>Tc&amp;hate...</translation>
```

Comment:
For french accelerators keys, "T" is a good key for "Tchate".
Please let it as it is.

Reply:
There is another item in that menu with T as the accelerator. One of them needed to be changed.

Reply:
Could you please tell me which one is the other ? I can't locate it.

Reply:
Pas de &tri des canaux

Reply:
I think it is not the same context. I will have a look to all the accelerator key for the next release as soon as possible.

Reply:
From my 3.8.2:
![image](https://user-images.githubusercontent.com/3224952/155508756-91ca3ce8-5310-4ba3-9b75-ca628e1cbded.png)

Reply:
I guess an alternative could be `&Pas de tri des canaux` allowing `&Tchate` within the menu, as that doesn't conflict with `&Tchate` in the main dialog.

Chat within the View menu was a new addition after 3.8.1

Reply:
Make "&P" for _premier_ (as that's what the option does); make "&C" for _canaux_ (as that's what the option does); make "&T" for _Tchate_ (as that's what the option does).  That way they're mnemonic.